src/Storage/Storage/Model/ResourceModel/PSPathAccessControl.cs

```diff
@@ -0,0 +1,98 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Storage.Shared.Protocol;
+using XTable = Microsoft.Azure.Cosmos.Table;
+using System.Collections.Generic;
+using System;
+using System.Linq;
+using Microsoft.Azure.Storage.File;
+using System.Net;
+using Microsoft.WindowsAzure.Commands.Common.Attributes;
+using Microsoft.Azure.Storage.Blob;
+
+namespace Microsoft.WindowsAzure.Commands.Storage.Model.ResourceModel
+{
+    /// <summary>
+    /// Wrapper class of Microsoft.Azure.Storage.Blob.PathAccessControlEntry
+    /// </summary>
+    public class PSPathAccessControlEntry
+    {
+
+        public bool DefaultScope;
+        public AccessControlType? AccessControlType;
+        public string EntityId;
+        public RolePermissions Permissions;
+
+        public PSPathAccessControlEntry(AccessControlType accessControlType, RolePermissions permissions, bool defaultScope = false, string entityId = null)
+        {
+            this.DefaultScope = defaultScope;
+            this.AccessControlType = accessControlType;
+            this.EntityId = entityId;
+            this.Permissions = permissions;
+        }
+        public PSPathAccessControlEntry(PathAccessControlEntry acl)
```

Comment:
Why `PathAccessControlEntry` is not `pace` or `ace`? What does `acl` mean?

Reply:
PathAccessControlEntry is just the type defined in XSCL, and we define psh class as PSPathAccessControlEntry, to align with it. Acl means: access control list.

Reply:
Got it